modules/virt-understanding-node-maintenance.adoc

```diff
@@ -1,25 +1,24 @@
 // Module included in the following assemblies:
 //
-// * virt/node_maintenance/virt-node-maintenance.adoc
-// * virt/node_maintenance/virt-setting-node-maintenance.adoc
+// virt/node_maintenance/virt-about-node-maintenance.adoc
 
-[id="virt-understanding-node-maintenance{context}"]
+[id="virt-understanding-node-maintenance_{context}"]
 = Understanding node maintenance mode
 
-Placing a node into maintenance marks the node as unschedulable and drains all 
-the virtual machines and pods from it. Virtual machine instances that have a 
-`LiveMigrate` eviction strategy are live migrated to another node without loss 
-of service. This eviction strategy is configured by default in virtual machine 
-created from common templates but must be configured manually for custom 
-virtual machines. 
+Nodes can be placed into maintenance mode using the `oc adm` utility, or using `NodeMaintenance` custom resources (CRs).
 
-Virtual machine instances without an eviction strategy will be deleted on the 
-node and recreated on another node. 
+Placing a node into maintenance marks the node as unschedulable and drains all the virtual machines and pods from it. Virtual machine instances that have a `LiveMigrate` eviction strategy are live migrated to another node without loss of service. This eviction strategy is configured by default in virtual machine created from common templates but must be configured manually for custom virtual machines.
+
+Virtual machine instances without an eviction strategy are shut down. Virtual machine instances with a `RunStrategy` of `Running` or `RerunOnFailure` are recreated on another node. Virtual machine instances with a `RunStrategy` of `Manual` are not automatically restarted.
```

Comment:
One more note.

VMI's don't have a run strategy. VM's do. I know that sounds a bit pedantic, but we need to be careful not to mix metaphors and stir confusion.

Reply:
Not pedantic at all, thanks I've just updated that.